src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py

```diff
@@ -268,36 +271,22 @@ def process_project(self):
         if (self.project_name is None) and (self.created_organization):
             self._create_project()
         elif self.project_name is None:
-            use_existing_project = prompt_y_n('Would you like to use an existing Azure Devops project? ')
+            use_existing_project = prompt_y_n('Would you like to use an existing Azure DevOps project? ')
             if use_existing_project:
                 self._select_project()
             else:
                 self._create_project()
         else:
             self.cmd_selector.cmd_project(self.organization_name, self.project_name)
 
+        self.logger.warning("To view your Azure DevOps project, "
+                            "please visit https://dev.azure.com/{org}/{proj}".format(
+                                org=self.organization_name,
+                                proj=self.project_name
+                            ))
+
     def process_yaml_local(self):
         """Helper to create the local azure-pipelines.yml file"""
-        # Try and get what the app settings are
-        with open('local.settings.json') as f:
```

Comment:
Remove unnecessary **local.settings.json** detection and **use_local_settings** detection